src/components/reusable/inlineCodeView/inlineCodeView.stories.js

```diff
@@ -0,0 +1,39 @@
+import { html } from 'lit';
+import './index';
+
+export default {
+  title: 'Components/Code View - Inline',
```

Comment:
Might be cleaner to nest both of these components under the same "Code View" sidebar item.

Reply:
please double-check that this was done properly

Reply:
✅